Dockerfile

```diff
@@ -1,20 +1,38 @@
-FROM debian:latest
+FROM debian:stretch
 MAINTAINER Ondrej Vasko
```

Comment:
MAINTAINER is deprecated

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated